src/components/Chatbar/components/ChatbarSettings.tsx

```diff
@@ -59,89 +62,105 @@ export const ChatbarSettings = () => {
     );
   }, [dispatch]);
 
-  return (
-    <div className="flex items-start gap-2 p-2 text-gray-500">
-      {conversations.length > 0 ? (
-        <Tooltip isTriggerClickable={true}>
-          <TooltipTrigger>
-            <button
-              className="flex h-[34px] w-[34px] cursor-pointer items-center justify-center rounded hover:bg-green/15 hover:text-green"
-              onClick={() => {
-                setIsOpen(true);
-              }}
-              data-qa="delete-conversations"
-            >
-              <IconTrashX size={24} strokeWidth="1.5" />
-            </button>
-          </TooltipTrigger>
-          <TooltipContent>{t('Delete all conversations')}</TooltipContent>
-        </Tooltip>
-      ) : null}
-
-      <Tooltip isTriggerClickable={true}>
-        <TooltipTrigger>
-          <Import
-            highlightColor={HighlightColor.Green}
-            onImport={(importJSON) => {
-              dispatch(
-                ConversationsActions.importConversations({ data: importJSON }),
-              );
-            }}
-            icon={<IconFileArrowLeft size={24} strokeWidth="1.5" />}
-          />
-        </TooltipTrigger>
-        <TooltipContent>{t('Import conversations')}</TooltipContent>
-      </Tooltip>
-
-      <Tooltip isTriggerClickable={true}>
-        <TooltipTrigger>
-          <button
-            className="flex h-[34px] w-[34px] cursor-pointer items-center justify-center rounded hover:bg-green/15 hover:text-green"
-            onClick={() => {
-              dispatch(ConversationsActions.exportConversations());
-            }}
-            data-qa="export-conversations"
-          >
-            <IconFileArrowRight size={24} strokeWidth="1.5" />
-          </button>
-        </TooltipTrigger>
-        <TooltipContent>{t('Export conversations')}</TooltipContent>
-      </Tooltip>
-
-      <Tooltip isTriggerClickable={true}>
-        <TooltipTrigger>
-          <button
-            className="flex h-[34px] w-[34px] cursor-pointer items-center justify-center rounded hover:bg-green/15 hover:text-green"
-            onClick={() =>
-              dispatch(
-                ConversationsActions.createFolder({ name: t('New folder') }),
-              )
-            }
-            data-qa="create-folder"
-          >
-            <FolderPlus height={24} width={24} />
-          </button>
-        </TooltipTrigger>
-        <TooltipContent>{t('Create new folder')}</TooltipContent>
-      </Tooltip>
+  const menuItems: DisplayMenuItemProps[] = useMemo(
+    () => [
+      {
+        name: 'Shared by me',
```

Comment:
use translations for names

Reply:
it's already done by `SidebarMenu` with `translation`